ToolBox/Polygon/PolygonDownloaderProgram.cs

```diff
@@ -54,35 +58,50 @@ public static void PolygonDownloader(IList<string> tickers, string securityTypeS
 
                 // Load settings from config.json
                 var dataDirectory = Config.Get("data-directory", "../../../Data");
-                var startDate = fromDate.ConvertToUtc(TimeZones.NewYork);
-                var endDate = toDate.ConvertToUtc(TimeZones.NewYork);
 
+                // Get hours
                 var marketHoursDatabase = MarketHoursDatabase.FromDataFolder();
 
                 // Create an instance of the downloader
                 using (var downloader = new PolygonDataDownloader())
                 {
+                    // Server side parameters for multithreaded simultaneous API connections
+                    ServicePointManager.DefaultConnectionLimit = 4;
```

Comment:
> ServicePointManager.DefaultConnectionLimit = 4;

Don't think this is required because we are not requesting in parallel?

Reply:
Yes, we need to request in Parallel to increase the bandwidth throughput.   The Polygon servers allow for 100 concurrent connections.

Reply:
Cool! So maybe the following foreach could be parallel for each? I was suggesting removing it because it's not required/used as is today

Reply:
The Tick class ( and perhaps other areas of the code ) are not thread safe.   I had initially used a Parallel ForEach in my earlier versions, but this broke the system.    I will remove it for now.

Reply:
This is a server wide configuration, so it is also important if I want to run two separate APIs at the same time, so I think I should leave it in here.

Reply:
> This is a server wide configuration, so it is also important if I want to run two separate APIs at the same time

Believe the setting affects only objects created in this process `Gets or sets the maximum number of concurrent connections allowed by a ServicePoint object.`  see https://docs.microsoft.com/en-us/dotnet/api/system.net.servicepointmanager.defaultconnectionlimit?view=net-5.0. Don't think it's required